### ISSUE TITLE
What to write into Remote Configuration Textbox?

### ISSUE THREAD

**Issue by notevil1337** [UnamSanctam/UnamWebPanel#283]
![Uploading Capture.PNG…]()

**Comment by UnamSanctam** [comment]
A URL to a "Remote Configuration" if you have one, for example on sites like pastebin https://github.com/UnamSanctam/SilentCryptoMiner/wiki/ETH-Miner#remote-configuration.

**Comment by notevil1337** [comment]
Yep, I don't have (auto-generated) config, should I write it on my own?

**Comment by UnamSanctam** [comment]
Sure that works fine, you can also test the JSON you write in any online JSON validator to make sure it's formatted correctly. Or you can enter the settings into the miner builder and go to the "JSON" tab for the auto-generated one.

**Comment by notevil1337** [comment]
That's great, I like the second idea lol
Thank you !

**Comment by notevil1337** [comment]
@UnamSanctam Sorry buddy, Last question!!!
Everything worked fine, the panel works also my test (VM) miner appeared but the (Hashrate is 0 H/s), should it be? And in general everything is as it should be?
![Capture](https://github.com/UnamSanctam/UnamWebPanel/assets/141959915/c0b5d493-eb27-4834-9b09-ac2334662a49)

**Comment by notevil1337** [comment]
I also cant see the miner on the xmr.2miners.com, When searching for my address, it throws an error "No miner data available

You need to submit at least a single share to see the stats."

**Comment by UnamSanctam** [comment]
Yes one share needs to be found first, what are the specifications on that VM? Does it have enough RAM and CPU resources to mine RandomX?

**Comment by notevil1337** [comment]
Lemme send you the ss

**Comment by notevil1337** [comment]
![image](https://github.com/UnamSanctam/UnamWebPanel/assets/141959915/9aeec1f4-25ec-492f-9b78-677df2726ce5)

**Comment by notevil1337** [comment]
Oh and ram is currently 5.28GB, Is it lower?

**Comment by UnamSanctam** [comment]
As long as there's at least like 2.3 GB of RAM free for RandomX it should work, and how many CPU cores did you give the VM?

**Comment by notevil1337** [comment]
![image](https://github.com/UnamSanctam/UnamWebPanel/assets/141959915/971d7159-868e-42c1-96a3-a871744868b4)
Only One..

**Comment by notevil1337** [comment]
Problem found? XD

**Comment by UnamSanctam** [comment]
Try increasing the cores to 2 (or more) and see, since that single core is probably already being used so it won't be used for mining.

**Comment by notevil1337** [comment]
So, I have to delete that one and build another one right?

**Comment by UnamSanctam** [comment]
You can stop the VM and then increase the CPU cores in the menu you took a screenshot of.

**Comment by notevil1337** [comment]
Alright, I thought I needed a new miner build

**Comment by notevil1337** [comment]
It works now bud..
But most people don't even know how or why to change the number of processors in their computer, and if I'm not mistaken it's set to 1 by default everywhere on Windows.

**Comment by UnamSanctam** [comment]
No, no real CPUs only have 1 core, only VMs that have been set to 1 core only have that.

**Comment by notevil1337** [comment]
Okay, Thank you so much <3
